### PR TITLE
add a dos gaming website

### PR DESCRIPTION
**Pull request by beingretrogamer**
This PR aims at adding a dos gaming website for playing dos games in browser without any setup.

### CODE REVIEW

File: docs/emulation.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ with a :gear:. You can get them
 - [Parallel Launcher](https://parallel-launcher.ca) - Nintendo 64 games
 - [:gear: DuckStation](https://www.duckstation.org) - PlayStation 1 games
 - [bsnes](https://github.com/bsnes-emu/bsnes) /
-  [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
```
Comment (Kazevic) [inline docs/emulation.md:66]: This line break might be an accident.

File: docs/emulation.md [hunk 2]
```diff
@@ -63,7 +63,8 @@ with a :gear:. You can get them
...
+- [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
```
Comment (Kazevic) [inline docs/emulation.md:66]: This line break might be an accident.

File: docs/emulation.md [hunk 3]
```diff
@@ -63,7 +63,8 @@ with a :gear:. You can get them
...
+- [WePlayDOS Games](https://weplaydos.games/) - Play 150+ classic dos games in browser (no-setup)
```
Comment (Kazevic) [inline docs/emulation.md:67]: You can change the description to something like "Web browser MS-DOS emulator" so that it's more consistent.

**Review by Kazevic — CHANGES_REQUESTED** [review]

File: docs/emulation.md (hunk 2 above)
Comment (beingretrogamer) [reply docs/emulation.md:66]: this is adding `-` in the last item.

File: docs/emulation.md (hunk 3 above)
Comment (beingretrogamer) [reply docs/emulation.md]: Done

File: docs/emulation.md (hunk 2 above)
Comment (Kazevic) [reply docs/emulation.md:66]: I mean, take a look at the line with melonDS and DeSmuME and you will see what's wrong with it. the `-` should be added to the new line, not to the unrelated Snes9x.

**Review by taskylizard — APPROVED** [review]

**Pull request merged**